ethcore/src/executive.rs

```diff
@@ -45,6 +44,11 @@ const UNPRUNABLE_PRECOMPILE_ADDRESS: Option<Address> = Some(ethereum_types::H160
 /// Precompile that can never be prunned from state trie (none)
 const UNPRUNABLE_PRECOMPILE_ADDRESS: Option<Address> = None;
 
+/// Gas per received storage key
```

Comment:
What about defining the value in `Schedule` https://github.com/openethereum/backport-3.1/blob/rakita/transaction_type/ethcore/vm/src/schedule.rs ?

Reply:
Moved to Schedule